scipy/stats/_entropy.py

```diff
@@ -46,20 +47,20 @@ def entropy(pk: np.typing.ArrayLike,
 
     Returns
     -------
-    S : {float, array_like}
+    H : {float, array_like}
         The calculated entropy.
 
     Notes
     -----
     Informally, the Shannon entropy quantifies the expected uncertainty
     inherent in the possible outcomes of a discrete random variable.
-
-    If messages consisting of sequences of symbols from a set S are to be
+    For example,
```

Comment:
Two things here: combined first two paragraphs so that Shannon entropy does not appear to be so much more important than relative entropy.

Also, "for example" because this is not the only interpretation.